### PR TITLE
docs: Add exercise for My own chain in IDA

### PR DESCRIPTION
**Pull request by xavierlepretre**
Documentation content update for "My own chain".

This PR adds pages before each section and transforms "My own chain" in a free-flow exercise.

### R1: Internal review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [x] requested
  - [x] completed

### R2: External review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R3: Internal QA review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R4: Final external QA review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### RC: Release Candidate

- [ ] Ready to be merged

### CODE REVIEW

File: academy/4-my-own-chain/create-handling-exer.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Exercise - Message Handler - Create and Save a Game Properly
+order:
```
Comment (xavierlepretre) [inline academy/4-my-own-chain/create-handling-exer.md:3]: @coldice How do we handle the ordering here?

Comment (coldice) [reply academy/4-my-own-chain/create-handling-exer.md:3]: It is ignored 😈

File: academy/4-my-own-chain-exer/game-fifo-exer.md [hunk 2]
```diff
@@ -25,15 +25,15 @@ Players can now reject games, so there is a way for stale games to be removed fr
...
+* How would you use Ignite CLI to locally run a one-node blockchain and interact with it via the CLI to see what you get?
 
 For now, don't bother yet with future ideas like wager handling.
 
 ## Next up
 
-When you are satisfied, or not, with what you did, have a look at a solution, which includes:
+When you are satisfied (or not) with what you have, look at some solutions which relate to:
 
-* a [FIFO](../4-my-own-chain/game-fifo.md),
```
Comment (AndrewB9lab) [inline academy/4-my-own-chain-exer/game-fifo-exer.md]: I made these changes to the link texts to "hide" the specific clues to the solutions that using "FIFO" etc represent - should be checked to make sure my alternatives are reasonable substitutes.

File: academy/4-my-own-chain-exer/week2-exercise.md [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
 ---
 
 # Exercise - Create an object
 
-We have prepared a small exercise on Hackerrank. It is so simple that it is basically meant to test your attendance. Before you head to it [here](TODO), you can read below the copy of what you will be asked.
+You are about to perform a small, simple exercise on Hackerrank, to confirm your attendance so far and check you're paying attention! Before you [try](TODO), here is a summary of what you will be asked to do.
 
 # The problem description
 
-This repository was created with Ignite CLI v0.17.3 and the following command:
```
Comment (AndrewB9lab) [inline academy/4-my-own-chain-exer/week2-exercise.md]: I reconsider - this change should probably be discarded.

File: academy/4-my-own-chain-exer/game-fifo-exer.md (hunk 2 above)
Comment (xavierlepretre) [reply academy/4-my-own-chain-exer/game-fifo-exer.md]: Here the FIFO is more about system performance. So "Secondary game structure for performance"?

Comment (AndrewB9lab) [reply academy/4-my-own-chain-exer/game-fifo-exer.md]: I would distill down to "System performance" maybe

File: academy/4-my-own-chain-exer/week2-exercise.md [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
 
 ## Student actions
 
-Your goal is to run a Ignite command to create a new Protobuf type that follows this description:
+Your goal is to run an Ignite command to create a new Protobuf type that follows this description:
 
-1. The name is `WorldParams`.
-2. It has 3 params exactly, no more, no less, and in this order exactly, with the Protobuf indices going from `1` to `3`:
+1. Its name is `WorldParams`.
+2. It has exactly 3 params, in exactly this order, with the Protobuf indices going from `1` to `3` respectively:
```
Comment (xavierlepretre) [inline academy/4-my-own-chain-exer/week2-exercise.md:34]:
```suggestion
2. It has exactly 3 parameters, in exactly this order, with the Protobuf indices going from `1` to `3` respectively:
```

File: academy/4-my-own-chain-exer/game-fifo-exer.md (hunk 2 above)
Comment (xavierlepretre) [reply academy/4-my-own-chain-exer/game-fifo-exer.md]: Ok

**Comment by CitMC** [comment]
Ready to be merged

**Pull request merged**